reactor-core/src/blockHoundTest/java/reactor/core/scheduler/BoundedElasticSchedulerBlockhoundTest.java

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright (c) 2011-Present VMware Inc. or its affiliates, All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *        https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package reactor.core.scheduler;
+
+import java.util.concurrent.CountDownLatch;
+import java.util.concurrent.TimeUnit;
+import java.util.concurrent.atomic.AtomicReference;
+
+import org.junit.Test;
+
+/**
+ * @author Simon Baslé
+ */
+public class BoundedElasticSchedulerBlockhoundTest {
+
+	//see https://github.com/reactor/reactor-core/issues/2143
+	@Test
+	public void shouldNotReportBlockingCallWithZoneIdUsage() throws Throwable {
+		AtomicReference<Throwable> errorRef = new AtomicReference<>(null);
+		CountDownLatch latch = new CountDownLatch(1);
+		Schedulers.single().schedule(() -> {
```

Comment:
FYI you can achieve the same with less code by submitting `FutureTask` (will re-throw from `.get`)

Reply:
good thing you made me rewrite the test, because I initially had it in the existing `ReactorBlockHoundIntegrationTest` class, which installs BlockHound, but I omitted that install part when splitting it out in its own testcase... so the test was a false positive!
fixed that and swithed to `FutureTask`